### DISCUSSION TITLE
How do I enable wumbo channels in the lightning implementation contained within older versions of BTCPay Server?

### DISCUSSION THREAD

**Discussion by clarketowson** [btcpayserver/btcpayserver#5711] [Technical Support]
Hi Everyone,

I want to enable wumbo channels on my BTCPay Server.  I have not seen this question asked before here.  I would like to know the config file location in the docker container and the command to include in it.

I understand that as of v23.11, this is the default (and thus, the option is ignored). Previously if you didn't specify this, channel sizes were limited to 16777215 satoshis.

I have an older version of BTCPay running which I will upgrade shortly but in the meantime it would be great if anyone who knows how to enable wumbo channels in older versions of lightning included within older versions of BTCPay that would be great.

Best Regards,

Clarke

Clarke Towson, BCMS (Bachelor of Computer & Mathematical Science)
CEO
INTJ Billing
 
m: +61 432 359 166
a: 7 Cullen Court Spotswood Victoria 3015 AUSTRALIA
w: https://intjbilling.com/
e: [clarketowson@intjbilling.com](mailto:clarketowson@intjbilling.com)

Bitcoin Lightning Network Node Name: CHILLYCALENDAR 
Node Public Key: 025124c73ef7ecf527e0114ead02a0cc6e3ecbc0c99474ee3f5506c4503b089693 
Status Page: https://intjbilling.com/INTJBillingStatusPage.php 

amboss.space: https://amboss.space/node/025124c73ef7ecf527e0114ead02a0cc6e3ecbc0c99474ee3f5506c4503b089693 
1ml.com: https://1ml.com/node/025124c73ef7ecf527e0114ead02a0cc6e3ecbc0c99474ee3f5506c4503b089693
Lightning network+: https://lightningnetwork.plus/nodes/025124c73ef7ecf527e0114ead02a0cc6e3ecbc0c99474ee3f5506c4503b089693

**Comment by dennisreimann** [comment]
The CLN conf should be editable similarily to editing the lnd.conf: https://docs.btcpayserver.org/FAQ/LightningNetwork/#how-to-edit-lndconf
